src/js/script.js

```diff
@@ -1,67 +1,108 @@
-function init() {
-    document.getElementsByClassName("timelineButton")[0].onclick = function() { apiCall(); }
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+const e = React.createElement;
+
+ReactDOM.render(
+	e("div", null, "Hello React!"),
+	document.getElementsByClassName("insert")[0]
```

Comment:
null, document.getElementsByClassName('insert') ... what if this is null similar on code below